css/home.css

```diff
@@ -348,6 +398,10 @@ Community Partnerships
 		text-align: center;
 		padding: var(--spacing-lg) var(--spacing-sm);
 	}
+
+	.community-card a {
+		max-width: 300px;
+	}
```

Comment:
learn more button should have its side padding adjusted and dependent on width in tablet and mobile mqs

Reply:
Much better, thanks! 🚀